naucse/templates/lesson.html

```diff
@@ -25,6 +25,15 @@
                 <hr>
             </header>
         {% endif %}
+        
+        {% if course is defined and not course.canonical %}
+            <div class="alert alert-info">
+                Toto je stránka lekce z kurzu, který probíhá nebo proběhl naživo s instruktorem.
+                {% if course.derives %}
+                    <a href="{{ canonical_url }}">Přejít na stejnou lekci v kurzu pro samouky</a>.
+                {% endif %}
+            </div>
+        {% endif %}
```

Comment:
Is this necessary? It seems to me that it would be annoying to see this all the time when actually attending a course.
Maybe the notice should go into session pages instead?